packages/application-shell/README.md

```diff
@@ -63,6 +63,34 @@ EntryPoint.displayName = 'EntryPoint';
 ReactDOM.render(<EntryPoint />, document.getElementById('root'));
 ```
 
+## Loading i18n messages with code splitting
+
+```js
+// define a function that accepts a language, and returns a promise.
+const applicationMessages = lang =>
```

Comment:
Nitpick: we can maybe name this variable `loadApplicationMessagesForLanguage`, to make it clear what it does